sources/views/contacts.js

```diff
@@ -20,40 +20,56 @@ export default class DataView extends JetView{
 				<div class='userImageWrapeSmall'>
 					<img class='userPhoto' src=${data.Photo || "http://milkyway.mie.uc.edu/cgdm/students/Male.png"}>
 				</div>
-				${data.FirstName || "nameless"}
-				${data.LastName || "empty"} <br>
-				${data.Email || "empty"}`;			
+				${data.FirstName || _("nameless")}
+				${data.LastName || _("empty")} <br>
+				${data.Email || _("empty")}`;			
 		};
 
 		let list = { 
 			rows:[
 				{cols:[
-					{rows:[
+					{css:"border", rows:[
 						{
+							view:"text", 
+							id:"filterForUser",
+							placeholder:_("type to find matching contacts"),
+							on:{
+								onTimedKeyPress:()=>{
+									let value = this.$$("filterForUser").getValue().toLowerCase();
+									this.$$("contactsList").filter(function(obj){
 
-							css:"border",
+										//It work bed because the most of time we find all contacts
+										// for(let key in obj){
```

Comment:
(1) > It work bed because the most of time we find all contacts
На самом деле там кидается ошибка при попытке привести к нижнему регистру число (id, StatusID). Вообще эти айдишки надо игнорировать при фильтрации, т.е. это не видимые для пользователя поля. 
(2) Лучше проверять наличие подстроки в значениях полей, а не её нахождение в начальной позиции - больше шансов найти искомый контакт